src/components/DestinationPopUp/DestinationPopUp.jsx

```diff
@@ -0,0 +1,57 @@
+import Styles from './DestinationPopUp.module.css';
+import World from '../../assets/Destination/World.jpg';
+import UnitedStates from '../../assets/Destination/UnitedStates.webp';
+import Spain from '../../assets/Destination/Spain.webp';
+import SouthEastAsia from '../../assets/Destination/SouthEastAsia.webp';
+import SouthAmerica from '../../assets/Destination/SouthAmerica.webp';
+import Protugal from '../../assets/Destination/Protugal.webp';
+import MiddleEast from '../../assets/Destination/MiddleEast.webp';
+import Italy from '../../assets/Destination/Italy.webp';
+import Greece from '../../assets/Destination/Greece.webp';
+import DestinationButton from './DestinationButton/DestinationButton';
+
+const DestinationPopUp = ({
+    title,
+    onClick,
+}) => {
+const destinations = [
+    {image:World, description:"I'm flexible"},
+    {image:UnitedStates, description:"United States"},
+    {image:Spain, description:"Spain"},
+    {image:SouthEastAsia, description:"Southeast Asia"},
+    {image:SouthAmerica, description:"South America"},
+    {image:Protugal, description:"Protugal"},
+    {image:MiddleEast, description:"Middle East"},
+    {image:Italy, description:"Italy"},
+    {image:Greece, description:"Greece"}];
+
```

Comment:
I noted that the Portugal, South America and Greece are not in the airbnb site. Not sure whether it should be corrected or not.

Reply:
Done